src/Components/Attachment.tsx

```diff
@@ -16,10 +16,12 @@ export const Attachment = connect(function Attachment({
   const [trigger, setTrigger] = useState<number>(0)
 
   useEffect(() => {
-    let timeoutId: NodeJS.Timeout
+    let timeoutId: NodeJS.Timeout | undefined
     dataBinaryToUrl(attachment).then(({ url, maxAge }) => {
       setBinaryUrl(url)
-      timeoutId = setTimeout(() => setTrigger(Date.now()), maxAge * 1000)
+      if (maxAge !== null) {
```

Comment:
* [ ] Doesn't solve the underlying issue of immediate execution if maxAge is too large.